github/gen-accessors.go

```diff
@@ -190,11 +192,12 @@ func (t *templateData) addIdent(x *ast.Ident, receiverType, fieldName string) {
 		zeroValue = "false"
 	case "Timestamp":
 		zeroValue = "Timestamp{}"
-	default: // other structs handled by their receivers directly.
-		return
+	default:
+		zeroValue = `nil`
```

Comment:
Style nitpick, use an interpreted string literal `"nil"` here rather than raw string literal, for consistency with the code above.